crates/primitives/src/result.rs

```diff
@@ -190,6 +186,21 @@ pub enum InvalidTransaction {
     BlobGasPriceGreaterThanMax,
 }
 
+impl<DBError> From<InvalidHeader> for EVMError<DBError> {
+    fn from(invalid: InvalidHeader) -> Self {
+        EVMError::Header(invalid)
+    }
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
```

Comment:
```suggestion
/// Errors related to misconfiguration of the  `BlockEnv`
#[derive(Debug, Copy, Clone, PartialEq, Eq)]
```

Reply:
resolved in https://github.com/bluealloy/revm/pull/731/commits/498d24c6329139248a664d9dbb11da6ace1760cc